src/index.html

```diff
@@ -11,12 +11,97 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:regular,500"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          href="/"
+          class="header__img"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
```

Comment:
The `href` attribute in the anchor tag should not have a trailing slash if it's meant to link to the home page, just use `href="#"`.